doc/devel/testing.rst

```diff
@@ -225,3 +225,50 @@ you may (rarely) need to build very old versions of Matplotlib.  The following
 constraints need to be taken into account:
 
 - Matplotlib 1.3 (or earlier) requires numpy 1.8 (or earlier).
+
+Testing regular installations of Matplotlib
```

Comment:
I think regular could be ambiguous. Maybe "non-development" ?

Reply:
I'm not comfortable defining this in terms of what this *not* is. I've rephrased to "Testing released versions of Matplotlib" and rewritten the following section a bit. I think "released version" is the correct qualifier, because only for releases you will be able to get the proper baseline images.

Reply:
i like "released versions" best 😄